.gitignore

```diff
@@ -86,8 +87,15 @@ contiki-cc2530dk.lib
 *.d71
 *.d81
 
+# Cooja Build Artifacts
+*.cooja
+!Makefile.cooja
```

Comment:
It really isn't needed. 

In my mind it makes sense to exclude it just to make the .gitignore match only what is needed and no more.  Presumably any file called Makefile.cooja is not a generated binary file but all other .cooja files are.